docs/elements/panel.mdx

```diff
@@ -0,0 +1,188 @@
+
+Panels are essential for manufacturing efficiency when producing multiple identical or different boards, as they allow all boards to be fabricated on a single panel and then broken apart.
+
+import CircuitPreview from '@site/src/components/CircuitPreview'
+
+<CircuitPreview defaultView="pcb" code={`
+
+  export default () => (
+    <panel width="100mm" height="80mm">
+      <board width="20mm" height="15mm">
+        <resistor resistance="1k" footprint="0402" name="R1" />
+      </board>
+      <board width="20mm" height="15mm">
+        <resistor resistance="1k" footprint="0402" name="R2" />
+      </board>
+    </panel>
+  )
+
+`} />
+
+
+## Panel Properties
+
+| Prop | Type | Description |
+| --- | --- | --- |
+| `width`, `height` | `Distance` | Define the panel's dimensions (e.g. "100mm", "4in"). |
+| `tabWidth` | `Distance` | Width of the tabs connecting boards (default: 2.5mm). |
+| `tabLength` | `Distance` | Length of the tabs connecting boards (default: 5mm). |
+| `boardGap` | `Distance` | Gap between boards in the panel (defaults to tabWidth). |
+| `mouseBites` | `boolean` | Enable mouse bite holes for easy board separation (default: true). |
+| `panelizationMethod` | `"tab-routing" \| "none"` | Method for connecting boards (default: "tab-routing"). |
+| `noSolderMask` | `boolean` | Prevent solder mask from being applied to the panel (default: false). |
+
+## Automatic Board Positioning
+
+When boards are added to a panel without explicit `pcbX` and `pcbY` positions, the panel automatically arranges them in a grid layout. The grid is sized to fit all boards efficiently.
+
+<CircuitPreview defaultView="pcb" code={`
+  export default () => (
+    <panel width="60mm" height="40mm">
+      {/* These boards will be automatically positioned in a grid */}
+      <board width="15mm" height="10mm">
+        <resistor name="R1" resistance="1k" footprint="0402" />
+      </board>
+      <board width="15mm" height="10mm">
+        <capacitor name="C1" capacitance="10uF" footprint="0603" />
+      </board>
+      <board width="15mm" height="10mm">
+        <led name="LED1" color="red" footprint="0603" />
+      </board>
+      <board width="15mm" height="10mm">
+        <resistor name="R2" resistance="10k" footprint="0603" />
+      </board>
+    </panel>
+  )
+`} />
+
+## Manual Board Positioning
+
+You can manually position boards within a panel using `pcbX` and `pcbY` properties. When any board has explicit positioning, automatic grid layout is disabled.
+
+<CircuitPreview defaultView="pcb" code={`
+  export default () => (
+    <panel width="50mm" height="30mm">
+      <board width="15mm" height="10mm" pcbX={-10} pcbY={0}>
+        <resistor name="R1" resistance="1k" footprint="0402" />
+      </board>
+      <board width="15mm" height="10mm" pcbX={10} pcbY={0}>
+        <capacitor name="C1" capacitance="10uF" footprint="0603" />
+      </board>
+    </panel>
+  )
+`} />
```

Comment:
This example in the preview looks symmetrical, it would be best to move them apart in a asymmetric way to put the point forward that these can be manual arranged.

<img width="998" height="586" alt="image" src="https://github.com/user-attachments/assets/a0275900-6342-496f-8454-e5ba0ff5aedd" />